build-watchman.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/sh
+
+set -eo pipefail
+
+WATCHMAN_VERSION="4.5.0"
+PCRE_VERSION="8.38"
+
+echo "*****************************"
+echo "building watchman ${WATCHMAN_VERSION}"
+echo "*****************************"
+echo
+set -x
+
+case $(uname -s) in
+  *Linux*) PLATFORM="linux"
+           ARCH=`uname -p`;;
+  *Darwin*) PLATFORM="mac";
+            ARCH=`sw_vers -productVersion | cut -f1,2 -d.`;;
+  *) echo "invalid platform"; exit 1;;
+esac
+
+DATE=`date -u +"%Y-%m-%dT%H:%M:%SZ"`
+DIRPATH=$(pwd -P)
+WATCHMAN_DEST_DIR="${DIRPATH}/build-support/bin/watchman/${PLATFORM}/${ARCH}/${WATCHMAN_VERSION}"
+BUILD_DIR="watchman_build.${DATE}"
+mkdir -p $BUILD_DIR
+
+PCRE_DIR="pcre-${PCRE_VERSION}"
+PCRE_TARBALL="${PCRE_DIR}.tar.gz"
+PCRE_URL="http://downloads.sourceforge.net/project/pcre/pcre/${PCRE_VERSION}/${PCRE_TARBALL}"
```

Comment:
Mind including a shasum check for this one? sourceforge ain't what it used to be.